### ISSUE TITLE
The latest workflow got strange O3 and PM2.5 predictions

### ISSUE THREAD

**Issue by JianpingHuang-NOAA** [NOAA-EMC/AQM#56]
I tested the latest workflow and noticed that the UFS-AQM got very strange behavior for both O3 nd PM2.5 predictions.  It seems that the model is not able to read emissions  correctly ?

![plot_grp1 timeseries OZONE 2022-07-01_12 2022-07-18_12 all CONUS](https://user-images.githubusercontent.com/54634403/214048453-aa3a0cd5-e55f-4a26-a858-91b69aba89a6.png)
![plot_grp1 timeseries PM2 5 2022-07-01_12 2022-07-18_12 all CONUS](https://user-images.githubusercontent.com/54634403/214048474-1e222640-2551-4e2d-adb8-be307bf29722.png)

@chan-hoo @BrianCurtis-NOAA @bbakernoaa

**Comment by JianpingHuang-NOAA** [comment]
My code: /lfs/h2/emc/physics/noscrub/jianping.huang/nwdev/packages/aqm.v7.0.53

Run dir:   /lfs/h2/emc/aqmtemp/para/tmp/run_fcst.id_1674395800_2022070412/   (Catcus)

**Comment by BrianCurtis-NOAA** [comment]
The code hasn't changed in a while. Something workflow related is my first assumption. Why is this run on Cactus if Cactus is the prod machine (currently)? I wonder if that has anything to do with it?

**Comment by JianpingHuang-NOAA** [comment]
It is on Dogwood, not Cactus.

**Comment by JianpingHuang-NOAA** [comment]
I have not seen upp.x generated in /scratch2/NCEPDEV/naqfc/Jianping.Huang/aqm.v7.0.53/exec  (Hera)

**Comment by chan-hoo** [comment]
upp.x is excluded in online-cmaq because run_post doesn't work with  ` post_config_fp="${HOMEdir}/sorc/AQM-utils/parm/postxconfig-NT-fv3lam_cmaq.txt"
` or varying fcst_len_hrs in the workflow. Do you use another 'postxconfig' file for run_post?

**Comment by JianpingHuang-NOAA** [comment]
It is fine with chemical fields, but not for meteorological fields. We
still rely on UPP to generate grib2 files for meteorological fields in case
we need to verify them.  I do not use another one postconfi file. We need
to figure out a way to fix the issue.

**Comment by chan-hoo** [comment]
Please let me know if you find a solution.

**Comment by chan-hoo** [comment]
As I mentioned above, you can't use the varying fcst_len_hrs option when run_post is true due to the structural issue on the workflow xml file.

**Comment by JianpingHuang-NOAA** [comment]
Post is not urgent for us at this moment.  The first priority for us is to make sure that the workflow is able to run the  forecast model successfully

**Comment by chan-hoo** [comment]
@JianpingHuang-NOAA, which task (which emission file) do you think causes this issue?

**Comment by drnimbusrain** [comment]
@chan-hoo @JianpingHuang-NOAA   I checked the NEXUS emissions from my latest authoritative [online-cmaq] run in community mode, and they look very reasonable.  E.g., NO2:  

![image](https://user-images.githubusercontent.com/26631222/214595165-49696109-8e16-4760-89a8-469f22f3fadd.png)

But, agreed that our new community runs the ozone and PM is not realistic anymore for some reason.  Looking into it as well.

**Comment by JianpingHuang-NOAA** [comment]
Here is my quick test results with the latest workflow and the
ufs-weather-model code. Now we can see the diurnal variation in surface o3
prediction. This means something was wrong when the code was merged into
the gfs-weather-model.  I am rerunning the simulations from July 4 to 15,
2022 and see whether we are able to reproduce c43 results .

[image: image.png]

**Comment by KaiWang-NOAA** [comment]
@JianpingHuang-NOAA The image can't be displayed? Also you meant the latest workflow and old ufs-weather-model code right?

**Comment by BrianCurtis-NOAA** [comment]
@rmontuoro Can you take a look and see if you have an idea on what might be happening?

**Comment by JianpingHuang-NOAA** [comment]
/lfs/h2/emc/ptmp/jianping.huang/emc.para/com/aqm/v7.0/c46.20230120/12  is
the location of the code that I checked out on Dogwood yesterday.
@***@***.*** ***@***.***>

commit 9afb82bfc00f4f143d0fba86fcdeb9d6327e76f2
Author: Raffaele Montuoro ***@***.***>
Date:   Thu Dec 22 17:58:27 2022 +0000

Raffale is not able to check it on WCOSS2

**Comment by BrianCurtis-NOAA** [comment]
I've chatted with Jianping, and I have asked him to revert the changes made in the commit below.

https://github.com/BrianCurtis-NOAA/AQM/commit/143cb570664786be69f5b19d56062490c5b5d752

Since he has stated that he can run OK with raffaele's fork/branch, then the only change I made beyond Raffaele's change with my merge into UFSWM and AQM was that commit listed above. If this works, I can worry about debug mode fixes later.
